src/main/proto/tech/figure/loan/v1beta1/mismo_loan.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package tech.figure.loan.v1beta1;
+
+option java_multiple_files = true;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+
+/*
+Loan originators may choose to simply upload a MISMO standardized loan package to store static loan data collected through
+their Loan Origination System. The current standard is MISMO 3.4.
+
+This data is generally not expected to change over time. The evolving state of the loan (commonly called "servicing data")
+is represented by the [`LoanState`](loan_state) proto.
+*/
+message MISMOLoan {
+  /*
+  Universal Loan Identifier (ULI) is a unique number made up of 23 to 45 characters that begins with the loan
+  originator's Legal Entity Identifier (LEI).
+  An originator's LEI can be found by searching the [GLEIF Website](https://search.gleif.org/#/search/).
+   */
+  string                            uli   = 1 [(validate.rules).string.min_len = 23, (validate.rules).string.max_len = 45];
+  // Byte array of the MISMO XML file
+  google.protobuf.BytesValue        data  = 2 [(validate.rules).message.required = true];
+  // Key-value map allowing originator to provide additional data
+  map<string, google.protobuf.Any>  kv    = 99;
```

Comment:
The more I think about this kv concept, it's nifty with a quirk. Consumers most of the time will probably only use String type to pass any custom data around. Otherwise, they can only use custom/strong type that are defined by the library. They can't really pass their own custom/strong type into our system as we won't be able to de-serialize it. Though, this kv prop is useful to use as a pass-through concept where they don't expect our system to read it but want to data come come back into their system.

Reply:
It's a challenge for sure to design a generic data model... in general, we expect that specific applications (like Portfolio Manager) will have specific requirements. They can submit PRs to this model if they want specific fields defined, or they could always publish their own standard of what they expect to be in an `kv` map.

Reply:
To pile on and clarify for future participants, it's expected that this repository will expand over time as new tangential use cases emerge. We need a contribution guide in this repository or can point to an existing one that explains the standard open source contribution process.